dev/devicelab/bin/tasks/run_release_test_macos.dart

```diff
@@ -2,122 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-import 'dart:convert';
-import 'dart:io';
-
 import 'package:flutter_devicelab/framework/devices.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
-import 'package:flutter_devicelab/framework/task_result.dart';
-import 'package:flutter_devicelab/framework/utils.dart';
-import 'package:path/path.dart' as path;
+import 'package:flutter_devicelab/tasks/run_tests.dart';
 
 /// Basic launch test for desktop operating systems.
 void main() {
-  task(() async {
-    deviceOperatingSystem = DeviceOperatingSystem.macos;
-    final Device device = await devices.workingDevice;
-    // TODO(cbracken): https://github.com/flutter/flutter/issues/87508#issuecomment-1043753201
-    // Switch to dev/integration_tests/ui once we have CocoaPods working on M1 Macs.
-    final Directory appDir = dir(path.join(flutterDirectory.path, 'examples/hello_world'));
-    await inDirectory(appDir, () async {
-      final Completer<void> ready = Completer<void>();
-      final List<String> stdout = <String>[];
-      final List<String> stderr = <String>[];
-
-      print('run: starting...');
-      final List<String> options = <String>[
-        '--release',
-        '-d',
-        device.deviceId,
-      ];
-      final Process run = await startFlutter(
-        'run',
-        options: options,
-        isBot: false,
-      );
-      int? runExitCode;
-      run.stdout
-        .transform<String>(utf8.decoder)
-        .transform<String>(const LineSplitter())
-        .listen((String line) {
-          print('run:stdout: $line');
-          if (
-            !line.startsWith('Building flutter tool...') &&
-            !line.startsWith('Running "flutter pub get" in ui...') &&
-            !line.startsWith('Resolving dependencies...') &&
-            // Catch engine piped output from unrelated concurrent Flutter apps
-            !line.contains(RegExp(r'[A-Z]\/flutter \([0-9]+\):')) &&
-            // Empty lines could be due to the progress spinner breaking up.
-            line.length > 1
-          ) {
-            stdout.add(line);
-          }
```

Comment:
FYI I removed this logic to skip standard output as it had no effect on the test. The `_findNextMatcherInList` helper used to verify console output skips lines in between the expected output.